src/libs/kata-types/src/config/runtime_vendor.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! A sample for vendor to customize the runtime implementation.
+
+use super::*;
+use crate::{eother, sl};
+
+/// Vendor customization runtime configuration.
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct RuntimeVendor {
+    /// Log level
+    #[serde(default)]
+    pub log_level: u32,
+
+    /// Prefix for log messages
+    #[serde(default)]
+    pub log_prefix: String,
+}
+
+impl ConfigOps for RuntimeVendor {
+    fn adjust_configuration(conf: &mut TomlConfig) -> Result<()> {
+        if conf.runtime.vendor.log_level > 4 {
```

Comment:
Should use some level const like debug/info instead of plain number.

Reply:
it will be fixed in next version